WordPress/WordPressTest/Tenor/TenorTests.swift

```diff
@@ -0,0 +1,30 @@
+import XCTest
+@testable import WordPress
+
+class TenorTests: XCTestCase {
+
+    func testTonerResponseParsing() {
+        let json = Bundle(for: TenorTests.self).url(forResource: "tenor", withExtension: "json")!
+        let data = try! Data(contentsOf: json)
+        guard let response = try? JSONDecoder().decode(TenorResponse.self, from: data) else {
+            XCTFail("Tenor response shuold be properly parsed")
+            return
+        }
+
+        XCTAssertEqual(response.next, "5", "Next page start offset should be correct")
+        XCTAssertEqual(response.results.count, 5, "Result count should be correct")
+
+        let media = response.results.first!
+
+        XCTAssertNotNil(media.gifs[.nanogif], "Nano gif should exist")
+        XCTAssertNotNil(media.gifs[.tinygif], "Tiny gif should exist")
+        XCTAssertNotNil(media.gifs[.mediumgif], "Medium gif should exist")
+        XCTAssertNotNil(media.gifs[.gif], "Normal gif should exist")
+
+        let gif = media.gifs[.gif]!
+
+        XCTAssertTrue(gif.dims[0] > 0 && gif.dims[1] > 0, "Gifs should have non-zero dimensions")
+        XCTAssertNotNil(gif.url, "Gifs should have a url")
+        XCTAssertNotNil(gif.preview, "Gifs should have a preview url")
```

Comment:
There's a lot of things being tested here. :)

What do you think about splitting those checks in different tests with better descriptions?

We can break it into distinct tests that check for:

* The properties (`title`, `id`)
* The `gifs` property and it's childs: `dims`, `url`, `preview`
* Maybe the `previewGif`, `largeGif`, `previewURL` and `staticThumbnailURL would make good tests too